ucm2/USB-Audio/Realtek-ALC1220-VB-Desktop-HiFi.conf

```diff
@@ -0,0 +1,60 @@
+SectionDevice."Speaker" {
+        Comment "Speakers"
+        Value {
+               PlaybackChannels "8"
+               PlaybackPCM "hw:ALC1220VBDT,0"
+               JackControl "Speaker Jack"
+               PlaybackMixerElem "Speaker"
+        }
+}
+
+SectionDevice."Headphone" {
+        Comment "Front Headphone"
+        Value {
+                PlaybackChannels "2"
+                PlaybackPCM "hw:ALC1220VBDT,1"
+                JackControl "Front Headphone Jack"
+                PlaybackMixerElem "Front Headphone"
+        }
+}
+
+SectionDevice."SPDIF" {
+        Comment "S/PDIF Out"
+        Value {
+                PlaybackPriority 1
+                PlaybackChannels "2"
+                PlaybackPCM "hw:ALC1220VBDT,2"
+                PlaybackMixerElem "IEC958"
+        }
+}
+
+SectionDevice."Line" {
+        Comment "Line In"
+        Value {
+                CaptureChannels "2"
+                CapturePCM "hw:ALC1220VBDT,0"
+                JackControl "Line Jack"
+                CaptureMixerElem "Line"
+        }
+}
+
+SectionDevice."Mic" {
+        Comment "Microphone"
+        Value {
+                CaptureChannels "2"
```

Comment:
Add priority 100 for mic1 and 200 for mic2 (front).

Reply:
In this case I think we should give Line-In also some priority, so I gave it priority of 100 and for these I gave 200 and 300 accordingly